vllm/model_executor/models/jamba.py

```diff
@@ -43,9 +43,14 @@
 
 @dataclass
 class MambaCacheParams:
-    is_prompt: bool = False
```

Comment:
Was this totally unused before??

Reply:
Yeah, since #8533, in #8533 we could drop `MambaCachePararms` as a whole, but forgot to do so, 
Good thing we need it here.